src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1560,8 +1560,11 @@ pub struct CreateTableFromSourceStatement<T: AstInfo> {
     pub constraints: Vec<TableConstraint<T>>,
     pub if_not_exists: bool,
     pub source: T::ItemName,
-    pub external_reference: UnresolvedItemName,
+    pub external_reference: Option<UnresolvedItemName>,
```

Comment:
Does this PR implement the thing where if no external reference is provided and it's unambiguous it's accepted? I didn't see any test like that added but this change looks like we accept it now?